### ISSUE TITLE
GPU Wait shows wrong data for some games in windowed mode

### ISSUE THREAD

**Issue by xMorioh** [GameTechDev/PresentMon#229]
Hi,

i've noticed that in some games like "Timberborn" as well as "Risk of Rain 2" i get wrong "GPU Wait" data in the following scenarios:
- when i set the game to Windowed mode
- when playing in Fullscreen and having the PresentMon UI rendering on top of the game

Specifically not being in those two scenarios above gives me the proper difference between GPU Busy and Frame Time.
Being in those two scenarios above i get 0.00 to 0.10ms readouts on "GPU Wait" while i can see that GPU Busy doesn't change and stays at around half the Busy time from the Frame Times due to the CPU bottleneck.

In other games like "Helldivers 2" i can render the game in windowed mode all i want but the "GPU Wait" data is correct and identical to Fullscreen mode.

In this image i switched from Windowed mode to Fullscreen playing "Timberborn", having the PresentMon UI detatched as separate Window.
![PresentMon_u82fL3VYNw](https://github.com/GameTechDev/PresentMon/assets/114238144/856cd618-c6d9-47e9-910f-0060bfc1e7db)

**Comment by JeffersonMontgomery-Intel** [comment]
Rendering in windowed mode (and using the PresentMon overlay in some situations/GPUs) will require extra GPU rendering/compositing work for other processes and that can change GPUWait for the target process.  It also depends on things like the presentation parameters used by the application, which will vary from game to game.

From the picture above, it looks like an increase in GPUWait going from windowed to fullscreen.  What are you expecting to happen, and why do you think that this data is incorrect?

**Comment by xMorioh** [comment]
Yeah that's kind of what i thought, i'm using PresentMon in detached windowed mode for that reason.

What i didn't know is that it can be normal in some games when rendering in windowed mode that GPU Wait appears to not directly equal to the time the GPU is waiting for the CPU to finish after the GPU has rendered the last frame which i guess make more sense when you look at presentation parameters or other things i don't know much about honestly, so thanks for your reply i'll see if i can dig deeper into this topic to understand it better :)
